doc/source/singlevisit.rst

```diff
@@ -0,0 +1,544 @@
+.. _singlevisit:
+
+==============================
+Single-visit Mosaic Processing
+==============================
+Standard calibration pipeline processing focuses on aligning data taken as 
+associations or as single exposures as closely to a standard astrometric coordinate
+frame as possible (see :ref:`runastrodriz-description` for full details).  
+This involves alignment of very few images all taken under
+as identical conditions as possible; namely, detector, filter, guide stars, 
+guiding mode, and so on.  These observations were intended to by the user to 
```

Comment:
These observations were intended **to** by the user to...

Remove the bold "to".